File: baekjoon/algo_10989.py
from sys import stdin
count = [ 0 for _ in range(10001)]
N = int(stdin.readline())
for _ in range(N):
    n = int(stdin.readline())
    count[n] += 1
    
for i in range(10001):
    while count[i] > 0:
        print(i)
        count[i] -= 1
        

File: baekjoon/algo_10989.cpp
#include<iostream>

int count[10001];
int N;

int main()
{
    scanf("%d", &N);
    
    for(int i=0; i < N; i++)
    {
        int c;
        scanf("%d", &c);
        count[c] += 1;
    }
    
    for(int i=1; i <= 10000; i++)
    {
        while(count[i] > 0)
        {
            printf("%d\n", i);
            count[i] -= 1;
        }
    }
        
}